### PR TITLE
OSDOCS-3673: Moved privatelink instructions

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
Enterprise-4.11

Issue:
[OSDOCS-3673](https://issues.redhat.com/browse/OSDOCS-3673)

Link to docs preview:

- ROSA: [Configuring a Private Cluster](https://50618--docspreview.netlify.app/openshift-rosa/latest/rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-private-cluster.html)
    - I edited a link in the [Configuring Private Connections ](https://50618--docspreview.netlify.app/openshift-rosa/latest/rosa_cluster_admin/cloud_infrastructure_access/rosa-configuring-private-connections.html)page. Not sure if this page should move as well, or if I need to note that the second bullet is in a different book.

Additional information:
Moved the PrivateLink instructions to under ROSA without STS.

### CODE REVIEW

**Comment by xueli181114** [comment]
LGTM

File: _topic_maps/_topic_map_rosa.yml [hunk 1]
```diff
@@ -162,6 +162,8 @@ Topics:
     File: rosa-installing-rosa
   - Name: Creating a ROSA cluster without AWS STS
     File: rosa-creating-cluster
+  - Name: Configuring a private cluster
```
Comment (jboxman-rh) [inline _topic_maps/_topic_map_rosa.yml:165]: Did we update the module includes listed with the new assembly name? `rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-private-cluster.adoc`

**Review by jboxman-rh — COMMENTED** [review]
@EricPonvelle, left a comment, thanks!

**Pull request merged**